### PR TITLE
Adds doctor's handbook

### PR DESCRIPTION
**Pull request by martinlyra**
Addressing #8.

What it does:
* Tells the user if the victim the book is used on is suffering toxin damage, blood loss, or malnourishment
* Gives descriptions on wounds, bruises, burns, and broken bones at each part of the body.
* It pronounces difference between alive and dead through flavor text.
* Currently does only work on humans

What is needed:
* Sprites

Other input is also appreciated.

### CODE REVIEW

**Comment by Chuckachu** [comment]
german medical book

Dirty version for German book?

**Comment by Chuckachu** [comment]
Possible Russian book?
russian book

**Comment by Chuckachu** [comment]
Perhaps change :
desc = "A book the size of your hand, containing a compact encyclopedia of the dark wonders of war - diseases, conditions, and documentation of all degrees of injury."

to

desc = "This book is the modern man's near complete treatise on all things medical related. With this in hand as a reference guide you can diagnose the aches and pains of any patient."

**Comment by Vivalas** [comment]
Internal bleeding seems to be impossible to diagnose unless your patient tells you, it should diagnose that as well.

**Comment by Vivalas** [comment]
Please wait until the current map changes in my PR go in before you map this in, since it will be a very minor map change here.

**Comment by martinlyra** [comment]
I don't plan on mapping them in, @Vivalas. I do plan on adding them to the starter loadout of medical roles, however.

File: Interstation-Two-WW2/code/modules/organs/wound.dm [hunk 1]
```diff
@@ -129,16 +132,22 @@
...
+			switch (damage_type)
+				if (BRUISE)
+					return prob(dam_coef*5)
+				if (BURN)
+					return prob(dam_coef*10)
+				if (CUT)
+					return prob(dam_coef*20)
 
-		return 0
+	return 0
```
Comment (Alexshreds) [inline Interstation-Two-WW2/code/modules/organs/wound.dm]: Why would you want this outside of the proc? The whole point is to return 0 if nothing else is returned

Comment (martinlyra) [inline Interstation-Two-WW2/code/modules/organs/wound.dm]: Suffice to say "inconsistent indentation"

**Comment by Wesreidau** [comment]
For the here and now, can we use generic book sprites? A simple green book should do. This is a hugely important tool that we're missing.

**Comment by martinlyra** [comment]
The book does indeed currently use generic book sprites. Black inventory/world icon. Reused the Bible for when held in hand

**Comment by martinlyra** [comment]
I've updated the code, but it remains largely untested and I've detected runtimes caused by it

**Comment by martinlyra** [comment]
I've - hopefully - tested the code thoroughly and the very specific runtimes seem to be gone now.
Removing the "do not merge" label now.

**Comment by Steelpoint** [comment]
This would be a great addition to medic.

**Pull request merged**